nixos/modules/services/continuous-integration/buildkite-agents.nix

```diff
@@ -1,8 +1,19 @@
 { config, lib, pkgs, ... }:
 
-with lib;
-
 let
+  inherit (lib)
+    mkOption
+    types
+    listToAttrs
+    concatStringsSep
+    mapAttrsToList
+    filterAttrs
+    literalExpression
+    literalMD
+    optionalString
+    mkRemovedOptionModule
+    ;
```

Comment:
IMO this is not improving anything. Either we prefix everything with lib. or keep it like it is.